Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -929,6 +962,19 @@ def delete_user(user_key):
     return 'User {} have been deleted.'.format(command_args['userKey'])
 
 
+def list_labels(user_key):
+    command_args = {
+        'userKey': user_key,
+    }
```

Comment:
not sure if i am missing anything but this variable isn't used at all

Reply:
Agreed, removing.